libs/src/services/creatorNode/CreatorNodeSelection.test.js

```diff
@@ -522,4 +522,39 @@ describe('test CreatorNodeSelection', () => {
     const healthyServices = [shouldBePrimary, shouldBeSecondary1, shouldBeSecondary2]
     healthyServices.map(service => assert(returnedHealthyServices.has(service)))
   })
+
+  it('does not always pick the same one with equivalency delta', async () => {
```

Comment:
this test isn't deterministic, but has a 9.53674316e-7 chance of failure.

felt this was more reasonable than trying to stub out the randomness, but happy to change if others feel strongly

Reply:
I'm cool with

Reply:
Not really, just that 1 in 1000 for a >= 50% chance seems a bit high. I do think reducing this chance would be a good idea. However, I don't think this is a priority